src/calculateRentalCost.js

```diff
@@ -29,6 +29,22 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const MIDDLE_TERM_DAYS = 3;
+  const MIDDLE_TERM_DISCOUNT = 20;
+  const LONG_TERM_DAYS = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const PRICE_PER_DAY = 40;
+  const totalRentalCost = days * PRICE_PER_DAY;
```

Comment:
I edited message, and renamed my proposition to 'basePrice', but you probably already changed it at the moment
change it once again please :)